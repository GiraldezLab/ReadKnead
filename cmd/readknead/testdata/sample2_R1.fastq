@HWI-ST1144:966:HKVXXXXX2:1:1101:1918:1964 1:N:0:AGTCAA
NATATACATTGAGTATGGTTATCATTATCATTTTCCCTCTTCATTTGCTCTTCAACGAAAGGTGAACAGGTGGAAC
+
#11ABDBDFHHHHIFHGGHGHHIEHHHGHIIIIIIIIEIGGGIHIIIBHHHGEIIIHHGIIIIIIIIHHGCHFGHI
@HWI-ST1144:966:HKVXXXXX2:1:1101:2813:1967 1:N:0:AGTCAA
NGCCAAACAGAAATACCAGCTGCACTGGGACCCAATTTCTCCTTTGGCTACCGATCCAGAAGTTAGATCGGAAGAG
+
#1BDFFDFHHHHHJJJJJJJJIJJJJJJJJJJJJJJJJJJJGIIIIIIHIJJJJJJJJJJJGHEHHHFFFFFFCFE
@HWI-ST1144:966:HKVXXXXX2:1:1101:3969:1976 1:N:0:AGTCAA
ATCCCGGTACGGGACTTATCAGTTTACCGTCACCACAGAAATTGTTTGACTATAAAAGACAATTCTGTGTAGTTTG
+
:==A+=DD;CCDFGB?EF:A?9CGFFFIECBFDDG;*?B?GF?4?D69?/BFCFI8CFIEA@CFFCDECEEB?A@D
@HWI-ST1144:966:HKVXXXXX2:1:1101:5724:1968 1:N:0:AGTCAA
GGGCCTCGAGAGCCACCATTCTGTAAAATTGAAGCACATTTTTCATTGTGTTTGGATCCGTCAGATCGGAAGAGCA
+
#1=DDDFFHGFHFIFGIJFGGGIJIJGGHHIIIIJJIDGHIJJJGGGIEGGGGHHIJHIIJIIGIHEECHHFFFFB
