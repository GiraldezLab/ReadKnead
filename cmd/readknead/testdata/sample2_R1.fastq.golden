@sample2.1
NATATACATTGAGTATGGTTATCATTATCATTTTCCCTCTTCATTTGCTCTTCAACGAAAGGTGAACAGGTGGAAC
+
#11ABDBDFHHHHIFHGGHGHHIEHHHGHIIIIIIIIEIGGGIHIIIBHHHGEIIIHHGIIIIIIIIHHGCHFGHI
@sample2.2
CGCCAAACAGAAATACCAGCTGCACTGGGACCCAATTTCTCCTTTGGCTACCGATCCAGAAGTT
+
CIEDFFHFHHHHHJJJJJJJJJJJJJJJJJJJJJJJJJJJJIJIIIIIHJJJJJJJJJJJJGHE
@sample2.3
ATCCCGGTACGGGACTTATCAGTTTACCGTCACCACAGAAATTGTTTGACTATAAAAGACAATTCTGTGTAGTTTG
+
:==A+=DD;CCDFGB?EF:A?9CGFFFIECBFDDG;*?B?GF?4?D69?/BFCFI8CFIEA@CFFCDECEEB?A@D
@sample2.4
GGGCCTCGAGAGCCACCATTCTGTAAAATTGAAGCACATTTTTCATTGTGTTTGGATCCGTC
+
HCGDDDFFHGGHFIFGIJGGGHIJIJGHHJIIIIJJIJGHIJJJGJJJGHGGHHHIJHIIJI
